tutorials/parquet-catalog-demos/euclid-q1-hats/4-euclid-q1-hats-magnitudes.md

```diff
@@ -215,6 +218,14 @@ Since the template-fit photometry is recommended for extended objects, we'll sep
 [Euclid Collaboration: Tucci et al., 2025](https://arxiv.org/pdf/2503.15306) defines point-like objects as having `MUMAX_MINUS_MAG < -2.5`.
 
 ```{code-cell} ipython3
+
+```
+
+```{code-cell} ipython3
+---
+jupyter:
+  source_hidden: true
```

Comment:
Just a FYI for the future, not something to address in this PR: sadly this is not the correct syntax, we get build warnings; but I already have an issue for cleaning things up so let's go ahead with it as is.

Reply:
https://github.com/IPAC-SW/ipac-sp-notebooks/issues/159 -- I usually open subissues there for things that are impacting all the JB2 repos we have